lib/pka_dev.c

```diff
@@ -637,7 +636,16 @@ static int pka_dev_set_ring_info(pka_dev_ring_t *ring)
 // Create shim. Set shim parameters and configure shim resources.
 // It returns 0 on success, a negative error code on failure.
 static int pka_dev_create_shim(pka_dev_shim_t *shim, uint32_t shim_id,
-                        uint64_t shim_base, uint64_t shim_size, uint8_t split)
+                        uint64_t shim_base, uint64_t shim_size,

```

Comment:
Please see comments in the driver source file

Reply:
Ack